Tests/TSCBasicTests/ProcessTests.swift

```diff
@@ -211,6 +211,28 @@ class ProcessTests: XCTestCase {
         }
     }
 
+    func testStdoutStdErrRedirected() throws {
```

Comment:
Are you planning to add the tests for the new functionality?

Reply:
wdym? this test case contains redirectStderr test cases by checking that stderr output is empty and that stdout contains stderr as well as stdout... what's missing?

Reply:
I mean tests for `OutputRedirections.stream(_, _, _)`, that it works.

Reply:
added those for streaming as well